examples/AccountManagement/GetAccountHierarchy.cs

```diff
@@ -0,0 +1,218 @@
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+            
+            try
+            {
+                codeExample.Run(new GoogleAdsClient(), customerId);
+            }
+            catch (GoogleAdsException adsException)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {adsException.Message}");
+                Console.WriteLine($"Failure: {adsException.Failure}");
+                Console.WriteLine($"Request ID: {adsException.RequestId}");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description =>
+            "This code example gets the account hierarchy of a specified manager account. " +
+            "If you don't specify a manager customer ID, the example will instead print the hierarchies " +
+            "of all accessible customer accounts for your authenticated Google account.";
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="googleAdsClient">The Google Ads client instance.</param>
+        /// <param name="customerId">The customer ID.</param>
+        public void Run(GoogleAdsClient googleAdsClient, long? customerId = null)
+        {
+             // googleAdsServiceClient = googleAdsClient.GetService(Services.V3.GoogleAdsService);
+            
+            // List of Customer IDs to handle.
+            ArrayList seedCustomerIds = new ArrayList();
+            
+            // If a Manager ID was provided above, it will be the only ID in the list.
+            // Otherwise, we will issue a request for all customers accessible by this authenticated Google account.
+            if(customerId.HasValue)
+            {
+                seedCustomerIds.Add(customerId);
+            }
+            else
+            {
+                Console.WriteLine("No manager customer ID is specified. The example will print the hierarchies of " + 
+                                  "all accessible customer IDs:");
+                CustomerServiceClient customerServiceClient = googleAdsClient.GetService(Services.V3.CustomerService);
+                string[] accessibleCustomers =  customerServiceClient.ListAccessibleCustomers();
+
+                foreach (string customerResourceName in accessibleCustomers)
+                {
+                    Customer customer = customerServiceClient.GetCustomer(customerResourceName);
+                    Console.WriteLine(customer.Id);
+                    seedCustomerIds.Add(customer.Id);
+                }
+                Console.WriteLine();
+            }
+            
+            // Creates a query that retrieves all child accounts of the manager specified in search calls below.
+            const string query = "SELECT customer_client.client_customer, customer_client.level,"
```

Comment:
If you look at https://github.com/googleads/google-ads-dotnet/blob/master/examples/BasicOperations/GetExpandedTextAds.cs, you'll see that the query is split a bit for readability. Each field is on its own line and the FROM and WHERE are a bit more split. Can we break this up a bit?

Reply:
Fixed.